skimage/data/__init__.py

```diff
@@ -259,3 +259,20 @@ def rocket():
 
     """
     return load("rocket.jpg")
+
+def _install_metadata_doc():
```

Comment:
I don't think we can do this, since it requires loading all images upon import.

Can we just hardcode the meta-data?